packages/opentelemetry/src/sampler.ts

```diff
@@ -72,6 +73,16 @@ export class SentrySampler implements Sampler {
       [SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE]: sampleRate,
     };
 
+    const method = `${spanAttributes[SemanticAttributes.HTTP_METHOD]}`.toUpperCase();
```

Comment:
Let's pull this into a separate PR!

Reply:
Yeah, that makes sense! Here: https://github.com/getsentry/sentry-javascript/pull/11467